src/server/scripts/Commands/cs_server.cpp

```diff
@@ -132,7 +132,10 @@ class server_commandscript : public CommandScript
     // Display the 'Message of the day' for the realm
     static bool HandleServerMotdCommand(ChatHandler* handler, char const* /*args*/)
     {
-        handler->PSendSysMessage(LANG_MOTD_CURRENT, Motd::GetMotd());
+        if (handler->IsHumanReadable())
```

Comment:
I can agree to that if you either scrap the current example or go with my initial suggestion (and dont implement the IsHumanReadable flag, always return true) to avoid having to change the same code twice